oauth2_http/javatests/com/google/auth/oauth2/ComputeEngineCredentialsTest.java

```diff
@@ -228,6 +242,20 @@ public void createScoped_defaultScopes() {
     assertEquals("foo", scopes.toArray()[0]);
   }
 
+  @Test
+  public void buildDefaultScoped_explicitUniverse() throws IOException {
+    ComputeEngineCredentials credentials =
+        ComputeEngineCredentials.newBuilder()
+            .setScopes(null)
+            .setUniverseDomain("some-universe")
+            .build();
+    ComputeEngineCredentials scopedCredentials =
+        (ComputeEngineCredentials) credentials.createScoped(null, Arrays.asList("foo"));
+
+    assertEquals("some-universe", scopedCredentials.getUniverseDomain());
+    assertTrue(scopedCredentials.isExplicitUniverseDomain());
+  }
```

Comment:
I think this test is fine, but perhaps we could make this one and the one above more generic. IIUC, the crux of the test is that it's testing that when `createScoped(...)` is called, the internal values are still the same (i.e. universe_domain).

Maybe a test where it sets stuff like quotaProjectId, universeDomain, accessToken, refereshMargin, scope, etc and ensures that everything is still the same except accessToken and scope. Test could be something like `buildDefaultScoped_fieldsRetained_accessTokenInvalidated()` for example (name could be better) or perhaps it can be merged with `buildScoped_shouldInvalidateAccessToken` from above.

Reply:
I merged these two new tests together. 
As of other properties, most of them are already covered in existing tests [here](https://github.com/googleapis/google-auth-library-java/blob/b995726fa657856f02ad474220f5734ae91f0331/oauth2_http/javatests/com/google/auth/oauth2/ComputeEngineCredentialsTest.java#L194-L220), and I added a new one verifying quotaProjectId.